utils/boto3_sessions.py

```
@@ -40,3 +40,43 @@ def initialize_health_client(aws_access_key, aws_secret_key, region):
+        # # Make a test API call to verify the connection
+        ses_client.list_verified_email_addresses()
+        print("Successfully connected to AWS SES service.")
+
+        return ses_client
+
+    except NoCredentialsError:
+        raise ValueError("AWS credentials are not provided or invalid.")
+    except PartialCredentialsError:
+        raise ValueError("Incomplete AWS credentials provided.")
+    except ClientError as e:
+        raise ValueError(f"Failed to connect to AWS Health service: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)